File: src/main/java/org/pyjjs/scheduler/core/api/impl/actors/task/messages/CheckOffersMessage.java
package org.pyjjs.scheduler.core.api.impl.actors.task.messages;

import akka.actor.ActorRef;
import org.pyjjs.scheduler.core.api.impl.actors.common.messages.Message;

public class CheckOffersMessage extends Message {
    public CheckOffersMessage(ActorRef sender) {
        super(sender);
    }
}

File: src/main/kotlin/org/pyjjs/scheduler/core/api/impl/actors/task/messages/CheckOffersMessage.kt
package org.pyjjs.scheduler.core.api.impl.actors.task.messages

import akka.actor.ActorRef
import com.google.common.collect.Lists
import org.pyjjs.scheduler.core.api.impl.actors.common.messages.Message

class CheckOffersMessage(sender: ActorRef) : Message(sender)
